controllers/users.js

```diff
@@ -416,13 +425,14 @@ module.exports = {
   _getContactParent: db.medic.get,
```

Comment:
Just future proofs the tests a tiny bit if we need to do something fancier rather than a simple get in the future.